torch/linalg/__init__.py

```diff
@@ -88,6 +88,41 @@
 Alias of :func:`torch.det`.
 """)
 
+slogdet = _add_docstr(_linalg.linalg_slogdet, r"""
+linalg.slogdet(input) -> (Tensor, Tensor)
+
+Calculates the sign and log absolute value of the determinant(s) of a square matrix or
+batches of square matrices :attr:`input`.
+
+Supports input of ``float``, ``double``, ``cfloat`` and ``cdouble`` datatypes.
+
+.. note:: When given inputs on a CUDA device, this function synchronizes that device with the CPU.
+
+.. note:: If ``input`` has zero determinant, this returns ``(0, -inf)``.
```

Comment:
This should describe what happens in the batched case, too. If `input` is reference in isolation it should be ":attr:\`input\`".

Reply:
Changed to 
```python
.. note:: For matrices that have zero determinant, this returns ``(0, -inf)``.
          If :attr:`input` is batched then the entries in the result tensors corresponding to matrices with
          the zero determinant have sign 0 and the natural logarithm of the absolute value of the determinant -inf.
```